.github/actions/validate-linux-packages/tests/test_validate_cli.py

```
@@ -878,6 +920,66 @@ def test_polythene_store_defaults_to_system_temp(
+
+
+def test_ensure_executable_store_raises_for_non_executable_filesystem(
+    validate_cli_module: object,
+    monkeypatch: pytest.MonkeyPatch,
+    tmp_path: Path,
+) -> None:
+    """A helpful error is raised when the directory is not executable."""
+    module = validate_cli_module
+    target = tmp_path / "store"
+
+    monkeypatch.setattr(module, "_supports_executable_stores", lambda base: None)
```

Ruff (0.14.0)

Unused lambda argument: `base`

(ARG005)